manifests/2.11.1/opensearch-dashboards-2.11.1.yml

```diff
@@ -9,47 +9,47 @@ ci:
 components:
   - name: OpenSearch-Dashboards
     repository: https://github.com/opensearch-project/OpenSearch-Dashboards.git
-    ref: '2.11'
+    ref: 989d8f41f37cca3275bf3fedc5c2057a717d1d64
   - name: functionalTestDashboards
     repository: https://github.com/opensearch-project/opensearch-dashboards-functional-test.git
-    ref: '2.11'
+    ref: 3b7c9ad26342ff7edb2a8086ec09fd39e5bcc5ec
```

Comment:
You can keep this as `2.11` as we do not actually bundle it

Reply:
ok sayali